### ISSUE TITLE
NoMethodError in 90.1-2019 air_loop_hvac_adjust_minimum_vav_damper_positions

### ISSUE THREAD

**Issue by eringold** [NREL/openstudio-standards#1502]
In this section of the code: https://github.com/NREL/openstudio-standards/blob/eb090c4ee0ba2e093c2fb6d36c3b459d97d39ceb/lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/ashrae_90_1_2019.AirLoopHVAC.rb#L618-L637

`oa_ctrl.maximumFractionofOutdoorAirSchedule` returns an OptionalSchedule. 
`if !max_oa_frac_sch.is_initialized` checks if `max_oa_frac_sch` is initialized. If it is, then attempts are made to cast `to_ScheduleRuleset`, `to_ScheduleConstant`, etc. However, `max_oa_frac_sch` is still an OptionalSchedule, for which `to_ScheduleRuleset` (etc) is not defined. 

`max_oa_frac_sch` should be 'got' as a Schedule after the `else`, e.g.: 

```
 else 
  max_oa_frac_sch = max_oa_frac_sch.get
   if max_oa_frac_sch.to_ScheduleRuleset.is_initialized 
     max_oa_frac_sch = max_oa_frac_sch.to_ScheduleRuleset.get 
     max_oa_frac_sch_type = 'Schedule:Year' 
   elsif max_oa_frac_sch.to_ScheduleConstant.is_initialized 
     max_oa_frac_sch = max_oa_frac_sch.to_ScheduleConstant.get 
     max_oa_frac_sch_type = 'Schedule:Constant' 
   elsif max_oa_frac_sch.to_ScheduleCompact.is_initialized 
     max_oa_frac_sch = max_oa_frac_sch.to_ScheduleCompact.get 
     max_oa_frac_sch_type = 'Schedule:Compact' 
   end 
 end 
```

or inline, e.g.:
```
 else 
   if max_oa_frac_sch.get.to_ScheduleRuleset.is_initialized 
     max_oa_frac_sch = max_oa_frac_sch.get.to_ScheduleRuleset.get 
     max_oa_frac_sch_type = 'Schedule:Year' 
   elsif max_oa_frac_sch.get.to_ScheduleConstant.is_initialized 
     max_oa_frac_sch = max_oa_frac_sch.get.to_ScheduleConstant.get 
     max_oa_frac_sch_type = 'Schedule:Constant' 
   elsif max_oa_frac_sch.get.to_ScheduleCompact.is_initialized 
     max_oa_frac_sch = max_oa_frac_sch.get.to_ScheduleCompact.get 
     max_oa_frac_sch_type = 'Schedule:Compact' 
   end 
 end 
```

**Comment by lymereJ** [comment]
Good catch! Do you want to submit a PR?

**Comment by eringold** [comment]
@lymereJ yes, will follow up shortly.

**Comment by lymereJ** [comment]
Thanks!

**Comment by mdahlhausen** [comment]
Fixed with #1503
